src/renderer/services/eth2/client/eth2ApiClient/cgEth2ValidatorApi.tsx

```diff
@@ -0,0 +1,102 @@
+import {ICGEth2ValidatorApi} from "../interface";
+import {
+    Attestation,
+    AttestationData,
+    AttesterDuty,
+    BeaconBlock,
+    CommitteeIndex,
+    Epoch,
+    ProposerDuty,
+    Root,
+    SignedAggregateAndProof,
+    Slot,
+    ValidatorIndex,
+} from "@chainsafe/lodestar-types";
+import {HttpClient} from "../../../api";
+import {IBeaconConfig} from "@chainsafe/lodestar-config";
+import {Json, toHexString} from "@chainsafe/ssz";
+import querystring from "querystring";
+
+export class CgEth2ValidatorApi implements ICGEth2ValidatorApi {
+    private readonly httpClient: HttpClient;
+    private readonly config: IBeaconConfig;
+    public constructor(config: IBeaconConfig, httpClient: HttpClient) {
+        this.config = config;
+        this.httpClient = httpClient;
+    }
+
+    public getAggregatedAttestation = async (attestationDataRoot: Root, slot: Slot): Promise<Attestation> => {
+        const query = querystring.stringify({
+            // eslint-disable-next-line camelcase,@typescript-eslint/camelcase
+            attestation_data_root: this.config.types.Root.toJson(attestationDataRoot) as string,
+            slot,
+        });
+        const url = `/eth/v1/validator/aggregate_attestation?${query}`;
```

Comment:
we probably should've extended our HttpClient so it can create clone with some url prefix. But I guess we can do that as separate issue

Reply:
if I change baseURL its gonna break storing network metrics, or gonna need to use some sort of "hacking" inside http client

Reply:
HttpClient is our implementation so he can hold base url separate from url prefix

Reply:
then best is to open separate task for this refactor

Reply:
open at #355